mkosi/__init__.py

```diff
@@ -1653,6 +1653,11 @@ def make_image(state: MkosiState, skip: Sequence[str] = [], split: bool = False)
     if state.config.sector_size:
         cmdline += ["--sector-size", state.config.sector_size]
 
+    if state.config.seed:
+        cmdline += ["--seed", str(state.config.seed)]
+    else:
+        cmdline += ["--seed", "random"]
```

Comment:
We can just make this a one liner `["--seed", str(state.config.seed) if state.config.seed else "random"]`

Reply:
And then you can also make it part of the `cmdline` declaration.